subtests/docker_cli/attach/attach.py

```diff
@@ -36,10 +36,34 @@ def wait_interactive_cmd(self):
                      % wait)
         time.sleep(wait)
 
+    def image_exist(self, image_name):
+        di = DockerImages(self)
+        image_list = DockerImages.get_dockerimages_list(di)
+        exist_status = DockerImages.filter_list_full_name(image_list,
+                                                          image_name)
+        return exist_status
+
+    def pull_image(self, image_name):
+        print "pull_image"
```

Comment:
Okay for debugging, but no ``print`` in final version.  If pulling is critical to the test, use ``self.loginfo()`` otherwise if it's only handy when something breaks, use ``self.logdebug()``.  Those methods make sure the logs are updated and they give control over verbosity level.